script.js

```diff
@@ -0,0 +1,73 @@
+function setBackgroundColor(heightScreen) {
+  if (heightScreen > 1000) {
+    $("#home-page").css("background-image", "none");
```

Comment:
Generally, it is preferred to keep all styling information in the css file so that we only have to check one place to know all the styles an application is using and to avoid having to comb through the logic in the js file to update styling information in the future. As such, it would be more maintainable here to assign a css class instead of the inline styles.

```css
#home-page.largeHeightScreen {
  backgroud-image: none;
  backgroud-color: rgb(133, 57, 98);
}
```

```js
if (heightScreen > 1000) {
  $('#home-page').addClass('largeHeightScreen');
}
```

Reply:
I remove this feature in full since it turned out that the background image works fine for all heights.